test/test_ftps_server.py

```diff
@@ -0,0 +1,83 @@
+import os
+import time
+from ftplib import FTP
+
+import pytest
+
+from domain.ftps_server import FTPsServer
+
+
+@pytest.fixture
+def ftps_server():
+    return FTPsServer(
+        "0.0.0.0", 21, 50, 5, "../cert/server.pem", "../cert/keyserver.pem", "./ftp_root"
```

Comment:
I guess wither we use ad-hoc test certificates we upload to the test dir, wither we have to generate them programmatically..

Reply:
I didn not find a way in python, we would need openssl on the test env... do we have?

@hexfati  please let's align on this before merge, all the rest is fine for me.

Reply:
You can add custom shell command in the env like this: https://github.com/stefanodellosa-personal/WADAS/blob/main/.github/workflows/test.yaml#L29C5-L31C19

It is possible to install packages as well. In this ase we might have to install packages and also generate the keys

Reply:
Documentation: https://docs.github.com/en/actions